src/gui/widgets/FxLine.cpp

```diff
@@ -23,22 +23,20 @@
  *
  */
 
-#include "FxLine.h"
```

Comment:
This `#include` should not be reordered. Besides being a [coding convention](https://github.com/LMMS/lmms/wiki/Coding-conventions), it checks that the header has all its dependencies declared.